### PR TITLE
Develop.redesign

### PR DESCRIPTION
**Pull request by Vladlukhanin**
_Make sure that you wrote the tests for your proposed changes and the existing test was success._

**Made/Proposed changes:**
_(Don’t use general words, describe changes in details)_
## 
## 

**How should this be manually tested?**

**Does the documentation need an update?**

### CODE REVIEW

File: app/images/qmweb-lounch-qblogo.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (dimaspirit) [inline app/images/qmweb-lounch-qblogo.svg]: Maybe you mean 'launch' instead of 'lounch', yep? It's about name of the file

File: app/index.html [hunk 2]
```diff
@@ -32,55 +33,63 @@
...
-      <div class="l-form l-flexbox l-flexbox_column is-hidden">
-        <button id="twitterDigitsSignUp" class="btn btn_form twitterDigits btn_qb j-twitterDigits">connect with phone number</button>
-        <a id="loginQB" class="text text_welcome" href="#">login by email or social</a>
-      </div>
-      <footer class="l-welcome-footer l-flexbox l-flexbox_column l-flexbox_flexbetween is-hidden">
-        <div class="l-welcome-footer-logo">
-          <a href="https://quickblox.com" target="_blank"><img src="images/logo-quickblox.svg" alt="QuickBlox"></a>
+        <footer class="l-welcome-footer l-flexbox l-flexbox_column l-flexbox_flexbetween is-hidden">
+          <button class="btn l-welcome_btn l-welcome_btn_green j-twitterDigits">connect with phone number</button>
+          <button id="loginQB" class="btn l-welcome_btn l-welcome_btn_white j-login_QB">login by email or social</button>
```
Comment (dimaspirit) [inline app/index.html]: Are you sure about this sentence 'login by email or social'?

File: app/index.html [hunk 3]
```diff
@@ -32,55 +33,63 @@
...
-          <div class="download">
-            <a class="download-app" href="https://play.google.com/store/apps/details?id=com.quickblox.q_municate" target="_blank"><img src="images/app-android.svg" alt="download Q-municate Android"></a>
-            <a class="download-app" href="https://itunes.apple.com/us/app/q-municate/id909698517?ls=1&mt=8" target="_blank"><img src="images/app-ios.svg" alt="download Q-municate iOS"></a>
+      <div class="l-welcome_container">
+        <div class="l-welcome_block_login l-welcome_logo_login"></div>
+        <div class="l-welcome_block l-welcome_text_logo_login">q-municate</div>
+        <div class="l-welcome_block l-welcome_text_login">Log In</div>
+        <form class="l-form l-form_login" action="#">
+          <span class="text_error">Error</span>
+          <div class="input-field login_email">
```
Comment (dimaspirit) [inline app/index.html]: Uses another class name 'login_email' because input has the same class name. It's confused me

File: app/index.html [hunk 4]
```diff
@@ -203,7 +221,7 @@ <h3 class="form-header">Reset your password</h3>
 
           <aside class="l-sidebar">
             <form id="searchContacts" class="l-search localSearch formSearch">
-              <input class="form-input-search" type="search" placeholder="Search">
+              <input class="form-input-search without_M" type="search" placeholder="Search">
```
Comment (dimaspirit) [inline app/index.html:192]: without_M - what is it means?

File: app/index.html [hunk 5]
```diff
@@ -254,7 +272,7 @@ <h3 class="l-list-header-title">Search results</h3>
               <div class="l-cap l-flexbox l-flexbox_column">
                 <span class="cap-text">Is anybody here?</span>
                 <img class="cap-anybody" src="images/anybody.png" alt="QuickBlox anybody">
-                <button class="search btn btn_add btn_invite"><img class="btn-icon btn-icon_invite" src="images/icon-invite.svg" alt="invite">Search for Friends</button>
+                <button class="search btn btn_add btn_invite without_M"><img class="btn-icon btn-icon_invite" src="images/icon-invite.svg" alt="invite">Search for Friends</button>
```
Comment (dimaspirit) [inline app/index.html]: alt="invite" === Search for Friends

File: app/scripts/main.js [hunk 6]
```diff
@@ -101,6 +101,12 @@ requirejs([
     // Application initialization
     $(function() {
         $.ajaxSetup({ cache: true });
+        // Materialize sdk
+        $.getScript('https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.6/js/materialize.min.js', function() {
```
Comment (dimaspirit) [inline app/scripts/main.js:110]: Write a comment in code why you made like this

File: app/styles/layouts.scss [hunk 7]
```diff
@@ -62,37 +62,123 @@
...
+  height: 488px;
+  border-radius: 4px;
+  background-color: #fff;
+  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, .1), 0 1px 2px 0 rgba(0, 0, 0, .2);
   position: relative;
-  width: 100%;
-  height: 40vh;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (dimaspirit) [inline app/styles/layouts.scss]: The best way is to use autoprefixer or you need add -webkit.
See caniuse.com

File: app/styles/layouts.scss [hunk 8]
```diff
@@ -62,37 +62,123 @@
...
+}
+
+.l-welcome_container.reset_pas {
+  height: 318px;
```
Comment (dimaspirit) [inline app/styles/layouts.scss]: We decided to use 4 spaces (https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/tree/master/web#general)

File: app/styles/layouts.scss [hunk 9]
```diff
@@ -62,37 +62,123 @@
...
+.l-welcome_block {
+  text-align: center;
+  vertical-align: middle;
+  position: relative;
+}
+  .l-welcome_logo {
+    width: 80px;
+    height: 80px;
+    margin: 0 auto;
+    background: image-url('qm-logo.svg') 50% no-repeat;
```
Comment (dimaspirit) [inline app/styles/layouts.scss]: What is it? 
`
 background: image-url
`

File: app/styles/layouts.scss [hunk 10]
```diff
@@ -62,37 +62,123 @@
...
   }
-  .l-welcome-header-text {
-    position: relative;
-    top: -1vh;
-    height: 5.2vh;
+    .l-welcome_text_logo {
+      width: 283px;
+      height: 49px;
+      margin: 35px auto 0;
+      font-family: Roboto;
```
Comment (dimaspirit) [inline app/styles/layouts.scss]: `
"Roboto", sans-serif;
`
uses like in framework

**Comment by dimaspirit** [comment]
upd hover/active state for btn/links.

**Comment by dimaspirit** [comment]
`
l-wrapper => width: 1000px;
`
It's bad idea. Made responsive

**Comment by dimaspirit** [comment]
Do you need this wrapper ?
http://joxi.ru/vAWDknyukPeNJr

File: app/images/qmweb-lounch-qblogo.svg (hunk 1 above)
Comment (Vladlukhanin) [inline app/images/qmweb-lounch-qblogo.svg]: loaded image, maybe mast be 'qmweb-launch-qblogo.svg'

File: app/index.html (hunk 4 above)
Comment (Vladlukhanin) [inline app/index.html:192]: It's class for reset Materialize.css styles (temporary)

File: app/index.html (hunk 5 above)
Comment (Vladlukhanin) [inline app/index.html]: Now it will be background image

**Pull request merged**